build.sh

```diff
@@ -20,7 +20,7 @@ function install_debrep {
 }
 
 function update_submodules {
-	git submodule update --init --recursive
```

Comment:
Perhaps consider tracking a remote branch in `.gitmodules` and using `git submodule update --init --recursive --remote`